spec/models/bale_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe Bale, type: :model do
```

Comment:
You are missing a serializer test here. It would help us to be sure that we are serializing the attributes we want to